packages/patterns-react/src/patterns/LeadSpaceCentered/index.js

```diff
@@ -0,0 +1,8 @@
+/**
+ * Copyright IBM Corp. 2016, 2018
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+export { default as LeadSpaceCentered } from './LeadSpaceCentered';
```

Comment:
@annawen1 Maybe I missed it, are you also exporting LeadSpaceCentered from /patterns?

Reply:
Gah. Yeah forgot that...adding it